src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1732,6 +1733,13 @@ public static HttpClientHandler CreateHttpMessageHandler(X509Certificate2 certif
                 ValidateClientKeyUsages(certificate);
             }
 
+#if NETCOREAPP
+            if (_getSocketsHttpHandler?.GetValue(httpMessageHandler) is SocketsHttpHandler socketsHttpHandler && socketsHttpHandler != null)
+            {
+                socketsHttpHandler.PooledConnectionLifetime = TimeSpan.FromMinutes(10);
```

Comment:
Should this be configurable via Conventions (NETCOREAPP only)? What should be the default value?